### ISSUE TITLE
Server fails to launch (but does not crash) when deeper darker is installed

### ISSUE THREAD

**Issue by p-cousin** [KyaniteMods/DeeperAndDarker#392]
**Minecraft Version**: 1.21.0
**Mod Version**: 1.3.2.1
**Mod Loader**: fabric 0.16.9

**Description**:
When I tried adding deeper and darker to my dedicated server (Nitroserv) the server got stuck on the reboot, without crashing. It just doesn't do anything (but in a solo copy of the world with the entire server modpack and the client modpack, everything works just fine).

**To Reproduce**:
1. Get a Nitroserv Minecraft server
2. Install Fabric and the server modpack
3. launch server : it works !
4. Add deeper and darker mod
5. reboot server : *gets stuck*

**Screenshots**:
Here's a screenshot of the server's console (for the whole console check the log file)
![Capture d'écran 2024-11-13 220413](https://github.com/user-attachments/assets/a57a7692-2524-439f-8c65-807dcce13153)

**Related files**:
Here are the logs and a list of the mods (the modpack is too large)
[client_modpack.txt](https://github.com/user-attachments/files/17881857/client_modpack.txt)
[latest.log](https://github.com/user-attachments/files/17881858/latest.log)
[server_mods+datapacks.txt](https://github.com/user-attachments/files/17881859/server_mods%2Bdatapacks.txt)

**Additional context**:
I've already contacted the Nitroserv support staff, and they're about as confused as I am so I thought I'd try my luck here... I've tried about everything I could with the mods, every time the server doesn't start it's when I have deeper and darker installed

**Comment by Pedro270707** [comment]
The log says nothing about Deeper and Darker or any other mod

**Comment by p-cousin** [comment]
> The log says nothing about Deeper and Darker or any other mod

yeah i can give you a log of the server starting properly with every other mod except deeper and darker if you'd like...

that's the weird part there's no error or anything, no clue about what causes the issue, but i added a batch of like 10 mods and the issue appeared, then i added them one by one, and deeper and darker was the only one that caused the issue. with or without the other mods
